src/data_vis.ts

```diff
@@ -90,7 +90,16 @@ class DataVisModel {
     }));
 
     private xAxisOptions = plottingVariables;
-    private disaggregationOptions = plottingVariables;
+    private disaggregationOptions = ko.computed(() => {
+        switch (this.currentPlot()) {
+            case "Impact":
```

Comment:
Is this case required when it is the same as the default case?

Reply:
Yes, strictly speaking this this is unnecessary.
I find it easier to follow switch statement when each option has its own block.

If you feel strongly I can refactor.